### ISSUE TITLE
Airbrake producing invalid times for event logging when using Rails 7.1.4 and up

### ISSUE THREAD

**Issue by gspugh-sb** [airbrake/airbrake#1266]
Airbrake's logging expects a time in seconds and has code in 13.0.4 that attempts to produce this time by dividing the Event time by MILLISECOND.

See:
```
def time
  # On Rails 7+ `ActiveSupport::Notifications::Event#time` returns an
  # instance of Float. It represents monotonic time in milliseconds.
  # Airbrake Ruby expects that the provided time is in seconds. Hence,
  # we need to convert it from milliseconds to seconds. In the
  # versions below Rails 7, time is an instance of Time.
  #
  # Relevant commit:
  # https://github.com/rails/rails/commit/81d0dc90becfe0b8e7f7f26beb66c25d84b8ec7f
  @rails_7_or_greater ? @event.time / MILLISECOND : @event.time
end
```

This actually results in invalid times when used together with **Rails 7.1.4** and higher. It produces times in **minutes** instead of **seconds**, resulting in errors such as:
`{"timestamp":1731421368307,"message":"E, [2024-11-12T14:22:48.307693 #62] ERROR -- : **Airbrake: time=\"1970-01-21 00:57:00 +0000 UTC\" is too far in the past"}
`

This is because starting with Rails 7.1.4, Rails is performing the conversion to seconds in `Event#time` (see: https://github.com/rails/rails/blob/v7.1.4/activesupport/lib/active_support/notifications/instrumenter.rb):
```
def time
  @time / 1000.0 if @time
end
```

Airbrake 13.0.1 returns `@event.time` from the `time` method.
13.0.2 and higher change that behavior to `@rails_7_or_greater ? @event.time / MILLISECOND : @event.time`

It looks like Airbrake needs a patch to change from `@rails_7_or_greater` to `@rails_7_1_4_or_greater` to work with newer versions of Rails.

In summary:
* Rails 7.1.3 with Airbrake 13.0.4 works (because **airbrake** performs the division to produce a time in seconds)
* Rails 7.1.4+ with Airbrake < 13.0.2 works (because **Rails** performs the division to produce a time in seconds)
* Rails 7.1.4+ with Airbrake 13.0.4 fails (because together they produce a time in minutes only)

**Comment by tomfast** [comment]
I'm also getting this error with Rails 7.2.2 and Airbrake version 13.0.1 and 13.0.4. Looks like a patch is on `master`: https://github.com/airbrake/airbrake/pull/1262
